### ISSUE TITLE
blurred text on r4

### ISSUE THREAD

**Issue by fedan** [OpenXRay/xray-16#113]
Hello!
Layers\xrRender\dxFontRender.cpp:120

``` c++
#ifndef USE_DX10
```

replace to

``` c++
#if !defined(USE_DX10) && !defined(USE_DX11) // && !defined(USE_OGL) // <- ???
```

and fix it.

я тестировал на dx10 с убраным "#ifndef    USE_DX10", т.к. карта gtx 275 у меня.
с выключеным текст мылит.

**Comment by fedan** [comment]
правильней так наверно:
```c++
#if !defined(USE_DX10) && !defined(USE_DX11) // && !defined(USE_OGL) // <- ???
```

**Comment by Xottab-DUTY** [comment]
There's the same issue, that already assigned to Swartz27: #53
There's commit by him in his fork with fix (but without "&& !defined(USE_OGL)": 66e95fad305f9805726d14c87663496fc477cc9a

**Comment by nitrocaster** [comment]
As said, see #53.
